components/stepper/stories/stepper.stories.js

```diff
@@ -1,9 +1,14 @@
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
 import { isFocused, isInvalid } from "@spectrum-css/preview/types";
-import { html } from "lit";
-import { styleMap } from "lit/directives/style-map.js";
 import { version } from "../package.json";
-import { Template } from "./template";
+import { 
+	Template,
+	SizingTemplate,
+	QuietTemplate,
+	DisabledTemplate,
+	DefaultTemplate,
+	StepperGroup,
```

Comment:
Same note here as for Steplist to use the Variants exported by the `stepper.test.js` file.